frontend/src/components/common/AlertNotification.tsx

```diff
@@ -99,32 +97,32 @@ export function PureAlertNotification({ checkerFunction }: PureAlertNotification
   }
 
   return (
-    <Box
+    <Alert
+      variant="filled"
+      severity="error"
       sx={theme => ({
         color: theme.palette.common.white,
+        background: theme.palette.error.main,
         textAlign: 'center',
         display: 'flex',
-        paddingTop: theme.spacing(1),
-        paddingBottom: theme.spacing(0.5),
+        paddingTop: theme.spacing(0.5),
```

Comment:
looks nice from out sync earlier, I would maybe try adding some break points for the smaller view so that it is stacked and not so squished

Reply:
Tweaked the padding on the right, should look more proportionate now